doc/common-issues.rst

```diff
@@ -0,0 +1,101 @@
+Frequently Encountered Issues
+=============================
+
+.. contents::
+
+Server reports wire version 5, PyMongo requires 6
+-------------------------------------------------
+
+When one attempts to connect to a <=3.4 version server, PyMongo will throw the following error:
+
+.. code-block::
+
+	>>> client.admin.command('ping')
+	...
+	pymongo.errors.ConfigurationError: Server at localhost:27017 reports wire version 5, but this version of PyMongo requires at least 6 (MongoDB 3.6).
+
+This is caused by the driver being too new for the server it is being run against.
+To resolve this issue either upgrade your database to version >= 3.6 or downgrade to PyMongo 3.x which supports MongoDB >= 2.6.
+
+
+'Cursor' object has no attribute '_Cursor__killed'
+--------------------------------------------------
+
+On versions of PyMongo <3.9, when supplying invalid arguments the constructor of Cursor,
+there will be two exceptions raised: AttributeError and TypeError. The AttributeError is not relevant,
+instead look at the TypeError for debugging information.
+
+.. code-block::
+
+	>>> Cursor(wrong=1)
+	Exception ignored in: <function Cursor.__del__ at 0x1048129d8>
+	...
+	AttributeError: 'Cursor' object has no attribute '_Cursor__killed'
+	...
+	TypeError: __init__() got an unexpected keyword argument 'wrong'
+
+To fix this, make sure that you are supplying the correct keyword arguments.
+In addition, you can also upgrade to PyMongo >=3.9, which will remove the spurious error.
+
+
+MongoClient fails ConfigurationError: Unknown option type_registry
```

Comment:
I would not consider this to be a common issue. I don't think the `type_registry` argument is particularly relevant to why this issue was viewed a lot. People are likely searching for "ConfigurationError: Unknown option <anything>" and ending up at the type_registry ticket. Let's document the error more generally with a made up or misspelled option instead of type_registry.